data/os/Debian.yaml

```diff
@@ -0,0 +1,10 @@
+---
+ca_cert::supported:             true
+ca_cert::trusted_cert_dir:      '/usr/local/share/ca-certificates'
+ca_cert::update_cmd:            'update-ca-certificates'
+ca_cert::cert_dir_group:        'staff'
+ca_cert::cert_dir_mode:         '2665'
```

Comment:
Where does this default come from?
On a fresh install of ca-certificates in `amd/debian:10`,`amd/debian:11` and `amd/debian:12` containers the default is
```
root@60422e36a680:/# ls -l /usr/local/share/
total 0
drwxr-xr-x. 2 root root 6 May  9 14:06 ca-certificates
```